### PR TITLE
fix indent_style

### LINKED ISSUE #36: softTabs gets overwritten by editors config
Hi,

if my editor.softTabs is set to false and i want to edit a yaml file (which uses spaces for indentation) and also have my editorconfig configured for yaml files it starts throwing tabs into it. I don't know it this is intended behaviour, but i don't think so.

https://github.com/sindresorhus/atom-editorconfig/blob/821dbc0835c3fe4a6ff673440430cd1e80c1052e/index.js#L23

commenting out the last part of the line fixed the problem.

thanks :)

### PR DESCRIPTION
**Pull request by t128**
fixes #36

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,15 +20,13 @@ function init(editor) {
 			return;
 		}
 
-		const isTab = config.indent_style === 'tab' || !editor.softTabs;
```
Comment (kevva) [inline index.js]: Isn't `const isTab = config.indent_style === 'tab';` enough?

Comment (t128) [inline index.js]: yeah, but this way it's more "expressive" and maybe feature proof (if case they add more options to the *indent_style* field).

Comment (kevva) [inline index.js]: I think it's pretty safe though. Not sure what more indent_styles there are?

Comment (t128) [inline index.js]: u can use special chars for indentation ;)
```js
/**/function test() {
/*=>*/return "test";
/**/}

**Comment by t128** [comment]
also there should be 3. states for *indent_styles*
-tab (use tabs)
-space (use spaces)
-undefined (use editors default config)

**Comment by kevva** [comment]
What's the editors default?

**Comment by t128** [comment]
the config at *Preferences* => *Soft Tabs*

**Comment by kevva** [comment]
Then I guess it's spaces so we can safely use the `isTab` variable anyway.

**Comment by t128** [comment]
No, it's not o.0

**Comment by kevva** [comment]
It it's soft tabs it's spaces?

**Comment by t128** [comment]
Yeah, but if it's turned off then it should use tabs (unless specified by editorconfig).

**Comment by sindresorhus** [comment]
// @bobrocke @ezekg @ismay

**Comment by ezekg** [comment]
:+1:

**Pull request closed without merging**